indy_client/test/cli/test_pool_upgrade_validation.py

```diff
@@ -1,44 +0,0 @@
-from copy import deepcopy
-
-from indy_node.test.upgrade.conftest import validUpgrade
-from indy_client.test.cli.constants import INVALID_SYNTAX, ERROR
-from indy_node.test.upgrade.helper import loweredVersion
-from plenum.common.constants import VERSION
-from plenum.common.util import randomString
-from indy_common.constants import JUSTIFICATION, JUSTIFICATION_MAX_SIZE
-
-
-def testPoolUpgradeFailsIfVersionIsLowerThanCurrent(
```

Comment:
This tests need to be migrated to SDK and moved to indy-node folder

Reply:
Restored. Will be integrated in future PRs